flutter/linux/wayland_shortcuts_inhibit.cc

```
@@ -0,0 +1,190 @@
+    g_signal_connect(widget, "focus-out-event",
+                     G_CALLBACK(on_window_focus_out), NULL);
+
+    // If window already has focus, create inhibitor now
+    if (gtk_window_has_toplevel_focus(GTK_WINDOW(widget))) {
+      inhibit_keyboard_shortcuts(GTK_WINDOW(widget));
+    }
+  }
+}
+
+// Public API: Initialize shortcuts inhibit for a sub-window
+void wayland_shortcuts_inhibit_init_for_subwindow(void* registry) {
```

Cppcheck (2.19.0)

[style] The function 'wayland_shortcuts_inhibit_init_for_subwindow' is never used.

(unusedFunction)